source/reference/nix-manual.md

```diff
@@ -5,31 +5,31 @@
 :hidden:
 
 Nix pre-release (development) <https://hydra.nixos.org/job/nix/master/build.x86_64-linux/latest/download>
-Nix 2.19 (latest) <https://nix.dev/manual/nix/2.19/>
-Nix 2.18 (rolling) <https://nix.dev/manual/nix/2.18/>
-Nix 2.18 (stable 23.11) <https://nix.dev/manual/nix/2.18/>
-Nix 2.13 (stable 23.05) <https://nix.dev/manual/nix/2.13/>
+Nix @latest@ (latest) <https://nix.dev/manual/nix/latest/>
+Nix @rolling@ (in rolling) <https://nix.dev/manual/nix/rolling/>
+Nix @stable@ (in stable @nixpkgs-stable@) <https://nix.dev/manual/nix/stable/>
+Nix @prev-stable@ (in stable @nixpkgs-prev-stable@) <https://nix.dev/manual/nix/prev-stable/>
 ```
 
 - [Nix pre-release](https://hydra.nixos.org/job/nix/master/build.x86_64-linux/latest/download)
 
   Reference documentation for the latest build from the `master` branch of the [Nix repository](https://github.com/NixOS/nix)
 
-- [Nix 2.19](https://nix.dev/manual/nix/2.19/)
+- [Nix @latest@](https://nix.dev/manual/nix/latest/)
 
   Reference documentation for the latest Nix release
 
-- [Nix 2.18](https://nix.dev/manual/nix/2.18/)
+- [Nix @rolling@](https://nix.dev/manual/nix/rolling/)
 
   Reference documentation for the Nix version shipped with the {term}`Nixpkgs` and {term}`NixOS` rolling release
 
-- [Nix 2.18](https://nix.dev/manual/nix/2.18/)
+- [Nix @stable@](https://nix.dev/manual/nix/stable/)
 
-  Reference documentation for the Nix version shipped with the current {term}`Nixpkgs` and {term}`NixOS` stable release
+  Reference documentation for the Nix version shipped with the current {term}`Nixpkgs` and {term}`NixOS` @nixpkgs-stable@ stable release
```

Comment:
Maybe we can remove all the "Reference documentation" prefix, since it's in every line.

Reply:
What do you suggest concretely? Just removing "reference documentation" would leave "Nix shipped with...", but it's the docs and not Nix itself.

Reply:
It would leave "For the Nix version shipped with..."